dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS/Utilities.pm

```diff
@@ -649,81 +649,78 @@ sub current_line_number {
   return $line_number;
 }
 
-=head2 C<Warn()>
 
-=over 4
 
-=item * Purpose
+=head2 Error handling methods
 
-Print warnings with line number details at the end.
+There are four main methods for reporting warnings and errors.
 
-=item * Arguments
+=over
 
-List of text to output.
+=item C<<$self->Warn(@messages)>>
 
-=item * Return Value
+This is equivalent to:
 
-None.
+  warn "@messages in foo.xs, line 123\n";
 
-=back
+The file and line number are based on the file currently being parsed. It
+is intended for use where you wish to warn, but can continue parsing and
+still generate a correct C output file.
 
-=cut
+=item C<<$self->blurt(@messages)>>
 
-sub Warn {
-  my ($self)=shift;
-  $self->WarnHint(@_,undef);
-}
+This is equivalent to C<Warn>, except that it also increments the internal
+error count (which can be retrieved with C<report_error_count()>). It is
+used to report an error, but where parsing can continue (so typically for
+a semantic error rather than a syntax error). It is expected that the
+caller will eventually signal failure in some fashion. For example,
+C<xsubpp> has this as its last line:
 
-=head2 C<WarnHint()>
+  exit($self->report_error_count() ? 1 : 0);
 
-=over 4
+=item C<<$self->death(@messages)>>
```

Comment:
This is not rendering correctly in POD formatters like `perldoc` or `pod2text`.  You are trying to say `$self->` but it's only coming out as `$self-`.  I believe the tip of the arrow needs to be an escaped greater-than sign.  The same problem occurs in these instances:
```
$ pod2text < dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS/Utilities.pm | ack '\$self-(?!>)'
        conditional nesting. In particular, it updates "@{$self-"{XSStack}}>
    "<$self-"Warn(@messages)>>
    "<$self-"blurt(@messages)>>
    "<$self-"death(@messages)>>
    "<$self-"WarnHint(@messages, $hints)>>
        Warn if the lines in "@{ $self-"{line} }> don't have balanced "#if",
```

Reply:
The issue is that the syntax to use the double-chevron POD escape is: `C<< ... >>` - there must be whitespace after the double chevron. Otherwise it is parsed as `C<...>death(@messages)>>`, where the part in the code escape is only `<$self-`.